src/checkstyle/checks.xml

```diff
@@ -164,5 +164,9 @@
                 jdk.nashorn.internal,
                 jdk.internal" />
         </module>
+
+        <module name="IllegalToken">
```

Comment:
Please move this above `IllegalImport`, so the illegal imports list stays at the bottom. Reasoning: the imports are mostly specific to Presto, but everything else is generic (for our coding style). At some point, I'd like to extract this out so we can reuse it for Airlift and other projects.